2022/form.py

```diff
@@ -74,7 +74,7 @@ def mustFile(self):
         """
         return self.must_file
 
-    def rowsum(self, rows):
+    def rowsum(self, rows, patterns=[]):
```

Comment:
I like this idea but I wonder if we could just make `rows` support regexs always and drop the need for an extra argument. That should be fine as long as none of the existing uses could be misinterpreted as regexes.

Reply:
That's much cleaner. Done..

> That should be fine as long as none of the existing uses could be misinterpreted as regexes.

To test this and others:
- I looked through a `git grep rowsum` for one year and didn't notice anything that could be misinterpreted as a regex
- I ran the last 3 years of my own taxes, and nothing changed

**Performance:**
Ran this 5 times each for my 2022 taxes, and took the average (variance was low):
```
    start = time.perf_counter()
    f = F1040(inputs)
    ca = CA540(inputs, f)
    end = time.perf_counter()
```

```
0.00046s original code
0.00667s matching each pattern individually
0.00517s using | to combine regexes
```

So there is a 11x performance decrease. But given the absolute numbers are so small, and given I can't imagine someone running a few thousand tax returns at once, I'd think this makes no difference practically. WDYT?